PRODUCTIONISATION.md

```diff
@@ -64,11 +64,10 @@ Ensuring that your project adheres to the following RFCs is a good first step as
 
 * 5.1 Continuous Delivery pipeline configured to:
   * [ ] 5.1.1 Automatically test pull requests
-  * [ ] 5.1.2 `master` branch to run linting and tests
+  * [ ] 5.1.2 `master` branch to run tests
   * [ ] 5.1.3 Automatic deploy to `staging` environment
   * [ ] 5.1.4 Manual step to deploy to `production` environment
-  * [ ] 5.1.5 Zero-downtime deployment step to production
-  * [ ] 5.1.6 Manual step to flip between blue/green environment
+   * [ ] 5.1.5 Zero downtime (e.g. blue/green environment) deployment step to production
```

Comment:
Let's not give `blue/green` as an example in this case, and keep this section a bit abstract.

There are plenty of ways to do Zero downtime

Reply:
It may be useful to leave it.
`blue/green` is just an example that help to clarify the "zero downtime" concept.

Reply:
Left `blue/green` as an example as suggested by Alessandro.